CONTRIBUTING.md

```diff
@@ -68,6 +68,12 @@ follow work through the issue/pull request process.
 5. **Open a pull request into `develop`.** Our main branch is `develop`, and
    this is where we want new work to be merged into. `master` is a release
    branch only.
+6. **Update `CHANGELOG.md` in your branch.** Add the pull-request number and 
+   link, and a brief, one line description under the next tag title. Make sure 
```

Comment:
Yeah, but the commits history is not really user friendly. I see commits more as a tool for the developers than a tool for the users. The `files changed` tab in the pull-request page is the best way to look, in one single place, at all the code in a feature/fix/change. Also, it becomes easy to see all the discussion involved in that change, and it's a nice, subtle way to give credit to the PR author/contributors, since their names are displayed there.